sw/airborne/firmwares/rotorcraft/stabilization/stabilization_attitude_quat_indi.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) Ewoud Smeur <ewoud_smeur@msn.com>
+ * MAVLab Delft University of Technology
+ *
+ * This control algorithm is Incremental Nonlinear Dynamic Inversion (INDI)
+ *
+ * This is a simplified implementation of the (soon to be) publication in the
+ * journal of Control Guidance and Dynamics: Adaptive Incremental Nonlinear
+ * Dynamic Inversion for Attitude Control of Micro Aerial Vehicles
+ *
+ * This file is part of paparazzi.
+ *
+ * paparazzi is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * paparazzi is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with paparazzi; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 59 Temple Place - Suite 330,
+ * Boston, MA 02111-1307, USA.
+ */
+
+#ifndef STABILIZATION_ATTITUDE_QUAT_INDI_H
+#define STABILIZATION_ATTITUDE_QUAT_INDI_H
+
+#include "firmwares/rotorcraft/stabilization/stabilization_attitude_ref_quat_int.h"
```

Comment:
firmwares/rotorcraft/stabilization/stabilization_attitude_rc_setpoint.c:374:51: error: 'stab_att_sp_euler' undeclared (first use in this function)